### ISSUE TITLE
Editor: text is stripped within code tags

### ISSUE THREAD

**Issue by shawnajean** [Automattic/wp-calypso#1109]
If you add this text in the editor, this sentence displays correctly:

```For example, <code>std::get&lt;0&gt;(tup)</code> retrieves the first element of a tuple.```

However, if you add this text instead the ```<0>``` gets stripped out.

```For example, <code>std::get<0>(tup)</code> retrieves the first element of a tuple.```

<img width="427" alt="0650ded4-7995-11e5-990c-7dd4bf87f512" src="https://cloud.githubusercontent.com/assets/775884/11504551/9401194e-980b-11e5-86c4-1b2ea90d5865.png">

This affects both the Calypso editor and WP.com's WP Admin editor. And both sentences publish correctly on my local install of core WP.

Test post on WP.com: https://testsjr1.wordpress.com/2015/10/23/testing-code-tags/

Test post on core install:
<img width="536" alt="99131c26-7be0-11e5-9b23-f5a4b49a97bf" src="https://cloud.githubusercontent.com/assets/775884/11504681/28431e5e-980c-11e5-8df3-373d430e20c7.png">

Ticket: 2195888-t

**Comment by Ipstenu** [comment]
Related. If I write a post in code tags, some things get encapsulated.

```
[code]
&lt;strong&gt;Rating:&lt;/strong&gt; {% if page.rating %}{% for i in (1..page.rating) %}&lt;i style=&quot;color:gold;&quot; class=&quot;fa fa-star&quot; name=&quot;star&quot;&gt;&lt;/i&gt;{% endfor %}{% if 5 &gt; page.rating %}{% assign greystar = 5 | minus: page.rating %}{% for i in (1..greystar) %}&lt;i style=&quot;color:grey;&quot; class=&quot;fa fa-star&quot; name=&quot;star&quot;&gt;&lt;/i&gt;{% endfor %}{% endif %}{% else %}&lt;em&gt;Not Available&lt;/em&gt;{% endif %}
[/code]
```

It was fine in the normal WP editor, though.

**Comment by hoverduck** [comment]
21Apr16 Bug Scrub

Confirmed, this is still an issue:
![1109-gh-wp-calypso](https://cloud.githubusercontent.com/assets/7233112/14713448/29270c50-07af-11e6-8994-3eeeb6e73e9a.gif)

**Comment by hoverduck** [comment]
26May16 Bug Scrub

Confirmed this is still an issue.

**Comment by alisterscott** [comment]
This only effects <tags> withing <code></code> blocks, which is also not supported in wp-admin.

[code][/code] blocks supports adding <tags> within them, and an issue where reloading a post with these would reformat these has been fixed in #8325 

I would suggest we close this as it's a limitation with wp-admin as well and the workaround to use [code][/code] blocks is acceptable.

**Comment by lancewillett** [comment]
> I would suggest we close this as it's a limitation with wp-admin as well and the workaround to use [code][/code] blocks is acceptable.

Done, agreed.
